GitUI/UserControls/FilterToolBar.cs

```diff
@@ -333,37 +333,34 @@ private void revisionGridFilter_FilterChanged(object? sender, FilterChangedEvent
             tsbShowReflog.Checked = e.ShowReflogReferences;
             InitBranchSelectionFilter(e);
 
-            tstxtRevisionFilter.Text = "";
```

Comment:
This still is necessary.

Reply:
Use case: Untick all checkboxes in the Filter dialog. Then the Filter textbox should be cleared. I.e. this must be done before the `if (any)`.

Reply:
I missed to save and commit in the first update.
Done after the check.

I want to keep the info in the box, otherwise will text entered but not Enter given always be cleared.

Reply:
This can be misleading because the pretty far away ![image](https://user-images.githubusercontent.com/36601201/193675589-b733e716-cf22-4f84-b8a2-ce0b0b841eeb.png) instead of ![image](https://user-images.githubusercontent.com/36601201/193675747-35ecd2bd-d122-4317-a9be-58b7e44143b9.png) is the only indication that the filter is not applied anymore.
(just discussing, not requesting a change)

Reply:
It is unchanged from now, if you type in the box the info is only used after Enter